packages/cli/openapi-ir-to-fern/src/__test__/__snapshots__/belvo.test.ts.snap

```diff
@@ -32411,16 +32402,7 @@ In our documentation example, we use \`{endpoint}\` as a placeholder value. In p
               "pattern": "[1-9]\\d{0,9}",
             },
           },
-          {
-            "type": "integer",
-            "validation": {
-              "exclusiveMax": undefined,
-              "exclusiveMin": undefined,
-              "max": undefined,
-              "min": 1,
```

Comment:
is losing this `min` value in the snapshots intentional ?

Reply:
This is a gap in how we handle validation rules on `long` types (and others). Now that we're correctly recognizing that this type is a `long`, we're dropping the rules. I'm planning to open an issue so that we can come back to this in a commit tailored to this.

Reply:
awesome -- mind linking the linear ticket once you file